### PR TITLE
fix issue #242

### PR DESCRIPTION
**Pull request by Student007**
#242

### CODE REVIEW

**Comment by Student007** [comment]
I'd played around with the .travis.yum that's the reason for the lot of commits. Finally it only depends on adding the wrapper to the dependencies

**Comment by Student007** [comment]
My Travis-Build runs well ! https://travis-ci.org/Student007/t2-cli

**Comment by LinusU** [comment]
Any specific reason why you aren't using `^` for depending on `wrappy`? That is the recommended way because http://semver.org/

**Comment by LinusU** [comment]
Also, that is a lot of commits :) you should probably squash that before merging :+1:

**Comment by Student007** [comment]
@LinusU yes I know the semver style. I wasn't clear in my head doing this ;-) ... I want to add ONLY the "1.0.1" version.

The reason is, I noticed not every programmer knows semver and the rule "Major, Minor, Fix" what makes me risky to use several "^compatible" versions. For that reasons I am used to use a exact versions.

Finally I changed it to compatible mode ;-)

If someone likes to take a look at my "t2 root" solution you may use my brunch "t2 root". But I haven 't done the tests right now. Also it is only installed on MacOSX. So I haven't checked the compatibility with windows ssh clients. I will do next days.

**Comment by Student007** [comment]
@LinusU the reason of so much minor commits had been the need to work directly with Travis-CI to figure out things. Maybe I misunderstand `should probably squash that before merging` - what do you mean ?

**Comment by LinusU** [comment]
Squashing is when you "squash" all commits related to a pull request into one commit. [Here is a stack overflow post on how to do it](http://stackoverflow.com/questions/14534397/squash-all-my-commits-into-one-for-github-pull-request)

**tl;dr** `git rebase -i HEAD~25` change `pick` to `squash` for all but first

I would also recommend you to work on a branch in your fork instead of on master, that allows you to keep you master in sync and then you do both squashing and "rebasing on master" with `git rebase master`.

**Comment by Student007** [comment]
@LinusU thanks for teaching this :smile:  ... didn't know that before.
For the "t2 root" I am currently working on a branch. Waiting for merging and then I will never ever use the master 
 :sunglasses: 
:pray:

**Comment by LinusU** [comment]
No problem, git is really cool :sunglasses: 

Seems like something went wrong thought, now there is 28 commits :laughing:

Just tried out the rebasing thing on my own computer but I'm getting lots of conflicts :confused: I think that the easiest thing is just to replace your `master` branch with the one from tessel; than create a new branch and apply the changes there.

Also, the dependencies should be in alphabetic order, please add them with `npm install --save _` instead of manually editing the package.json file :+1: 

The following commands should fix it all :)

```bash
# Fetch code from official repo
git remote add upstream git@github.com:tessel/t2-cli.git
git fetch upstream master

# Make their master your master
git reset --hard upstream/master
git push --force

# Make new branch
git checkout -b dependency-fix

# Add new dependencies
npm install --save wrappy
npm install --save terminal-menu

# Make commit
git add package.json
git commit -m "package: add missing dependencies"

# Push it 🙌
git push -u origin dependency-fix
```

**Comment by Student007** [comment]
@LinusU it looks like I have too learn some git.... thanks for this lesson :+1:

**Comment by LinusU** [comment]
No problem! We are all new to git at one point :) feel free to ping me if you ever need more help :+1:
